integration/containerd/cri/integration-tests.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+#
+# Copyright (c) 2017-2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+
+# Runtime to be used for testing.
+RUNTIME=${RUNTIME:-kata-runtime}
+readonly runtime_bin=$(command -v "${RUNTIME}")
+
+CRITEST=${GOPATH}/bin/critest
+
+#containerd config file
+readonly tmp_dir=$(mktemp -t -d test-cri-containerd.XXXX)
+export REPORT_DIR="${tmp_dir}"
+CONTAINERD_CONFIG_FILE="${tmp_dir}/test-containerd-config"
+
+
+info() {
+	echo -e "INFO: $*"
+}
+
+
+cleanup() {
+	rm -rf "${tmp_dir}"
+}
+
+err_report() {
+	echo "ERROR: contaienrd log :"
+	echo "-------------------------------------"
+	cat "${REPORT_DIR}/containerd.log"
+	echo "-------------------------------------"
+}
+
+trap err_report ERR
+trap cleanup EXIT
+
+info "Stop crio service"
+
+systemctl is-active --quiet crio && sudo systemctl stop crio
+
+info "testing using runtime: ${runtime_bin}"
+
+# make sure cri-containerd test install the proper critest version its testing
+rm -f "${CRITEST}"
+
+
+cri_containerd_repo="github.com/containerd/cri"
+
+pushd "${GOPATH}/src/${cri_containerd_repo}"
+cat > "${CONTAINERD_CONFIG_FILE}" << EOT
+[plugins]
+  [plugins.cri]
+    [plugins.cri.containerd]
+      [plugins.cri.containerd.default_runtime]
+        runtime_engine = "${runtime_bin}"
+EOT
+
+info "Running cri-tools"
+sudo -E PATH="${PATH}" \
+	FOCUS="runtime should support basic operations on container" \
+	REPORT_DIR="${REPORT_DIR}" \
+	CONTAINERD_CONFIG_FILE="$CONTAINERD_CONFIG_FILE" \
+	make test-cri
+
+info "Running test-integration"
+
+passing_test=(
+                TestClearContainersCreate
+                TestContainerStats
+                TestContainerListStatsWithIdFilter
+                TestContainerListStatsWithSandboxIdFilterd
+                TestContainerListStatsWithIdSandboxIdFilter
+                TestDuplicateName
+                TestImageLoad
+                TestImageFSInfo
+                TestSandboxCleanRemove
+             )
+
+for t in "${passing_test[@]}"
+do
+sudo -E PATH="${PATH}" \
```

Comment:
- Missing indent.
- If Fedora doesn't look for `/usr/local`, couldn't you do the following to avoid the `cp` above?:
  ```
  # Add local bin directory (required for sudo on Fedora)
  sudo -E PATH="${PATH}:/usr/local/bin" ...
  ```

Reply:
@jodh-intel  fixed :D